custom_conf.py

```diff
@@ -165,7 +165,9 @@
 # pyspelling, sphinx, sphinx-autobuild, sphinx-copybutton, sphinx-design,
 # sphinx-notfound-page, sphinx-reredirects, sphinx-tabs, sphinxcontrib-jquery,
 # sphinxext-opengraph
-custom_required_modules = []
+custom_required_modules = [
+    'watchfiles',
```

Comment:
This should go here instead: 
https://github.com/canonical/sphinx-docs-starter-pack/blob/4de5fb6905d96a63ad424d1b8283ac23c5b4bbf6/.sphinx/build_requirements.py#L85